src/check.c

```diff
@@ -9,7 +9,9 @@
 #include <stdlib.h>
 #include <string.h>
 #include <ctype.h>
+#ifndef USE_CRACKLIB
```

Comment:
And then everywhere replace `ifndef USE_CRACKLIB` (which is wrong anyway) with `ifdef HAVE_CRACK_H`.